### DISCUSSION TITLE
厦门大学怎么样啊？

### DISCUSSION THREAD

**Discussion by criphc** [CollegesChat/university-information#1] [Q&A]
如题。

**Comment by senior233** [comment]
翔安校区周边的环境肯定没有思明发达 但宿舍图书馆环境比思明好  上床下桌独立卫浴  上下学有共享电车  思明的饭好吃 
 另外  虽然金融是厦大品牌专业 但不见得就好 其他的就差   学校环境很舒服  大部分老师都很好  来了就会爱这个地方
